classes/classroom.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'student'
+
+class Classroom
+  attr_accessor :label, students
+  attr_reader :student
```

Comment:
> * [ ]  The variable you were requested to access , was `:students`, and Not `:student`, Kindly update that

- [OPTIONAL] While I understand that the previous reviewer was trying to tell you to add s to the student variable you created, I feel it is not needed, I mean the whole of the attr_reader because the attr_accessor is a shortcut for the getter and setter.